prow/config.yaml

```diff
@@ -186,6 +186,34 @@ presubmits:
     context: pull-kubernetes-federation-e2e-gce
     rerun_command: "@k8s-bot pull-kubernetes-federation-e2e-gce test this"
     trigger: "@k8s-bot (pull-kubernetes-federation-e2e-gce )?test this"
+    spec:
+      containers:
+      - args:
+        - --pull=$(PULL_REFS)
+        - --repo=k8s.io/kubernetes
+        - --git-cache=/root/.cache/git
+        - --clean
+        - --json
+        - --timeout=90
```

Comment:
you'll need `--upload=gs://kubernetes-jenkins/pr-logs` or else the default is `gs://kubernetes-jenkins/logs`